api/types/constants.go

```diff
@@ -440,6 +440,12 @@ const (
 	// that the resource originates from.
 	OriginLabel = TeleportNamespace + "/origin"
 
+	// CreatorTypeLabel is a resource metadata label name used to identify the creator type.
+	CreatorTypeLabel = TeleportNamespace + "/creator_type"
```

Comment:
That RFD covers tagging of resources created in our Cloud accounts for accounting purposes, but this will get deployed to a user's account. So I really don't see a reason to try and use the same set of labels. I'd just remove "creator_type" tbh.

Reply:
> this will get deployed to a user's account

We are teleport users.  It is likely this will get deployed to our account.

> What do we need "creator type" label for? It's a bit confusing, what are the other creator types?

This is to distinguish resources created by teleport vs resources created by terraform, (maybe) tenant-operator, and hand provisioned by users in the UI.

If we're going to have teleport differ from our other internal tooling, it'd be nice if end users could pass tags through, so we can make teleport behave consistently.

Perhaps that is the right approach? If we go this route, it'd be good to avoid the tag namespaces we already use for other purposes internally.

Reply:
I understand it might get deployed to our account, but 99% of the time it will get deployed to user account and I don't understand why we need to force our internal tagging scheme onto users.

Passing tags sounds like overkill.

Reply:
> I don't understand why we need to force our internal tagging scheme onto users.

:shrug: In lieu of passing tags, we're forcing a tagging scheme onto users whether it is consistent with our internal tagging scheme or not.  I'm fine if we punt on this -- passing tags is probably better for all users anyhow, and we can deal with that as its own request.

Reply:
I'm going to remove the `creator_type` and keep the others:
```
teleport.dev/creator: <clusterName>
teleport.dev/origin: aws-oidc-integration
teleport.dev/integration: <integrationName>
```